chain/cmd/bandsv/main.go

```diff
@@ -146,6 +169,26 @@ func handleRequestData(c *gin.Context) {
 
 	params := respParams.Result
 
+	if reqType == Unconfirmed {
+		// TODO: Replace by `BandStatefulClient` after implementation finished.
+		txr, err := bandProvider.SendTransaction(
+			[]sdk.Msg{zoracle.NewMsgRequest(req.CodeHash, params, 10, bandProvider.Sender())},
+			0, 20000000, "", "", "",
+			flags.BroadcastAsync,
+		)
+		if err != nil {
+			c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
+			return
+		}
+
+		c.JSON(200, OracleRequestUnconfirmedResp{
+			TxHash: txr.TxHash,
+		})
+		return
+	}
+
+	fmt.Println("After unconfirmed")
```

Comment:
If this line want to make other understand about code, make it to comment seem more make sense